docs/sphinx/intro.rst

```diff
@@ -111,6 +113,13 @@ Note that the level of readiness is not necessarily identical in both Peewee and
             <td class="warning"></td>
             <td align="center"><a class="glyphicon glyphicon-download-alt" href="_static/schema_graphs/auto/sdss5db.targetdb.pdf"></a></td>
         </tr>
+        <tr>
+            <td class="active">archive</td>
+            <td class="active">sas</td>
+            <td class="danger"></td>
+            <td class="success"></td>
+            <td align="center"><a class="glyphicon glyphicon-download-alt" href="schema/archive/archive_sas.pdf"></a></td>
```

Comment:
When the docs build this link is broken. If you want to add your own PDF do so in `docs/sphinx/_static/schema_graphs`.

Reply:
I fixed this link to point to the proper location.  The `_static/schema_graphs` directory is not committed to the repo.  It seems this directory is only set up for auto-generating schema.  Unless we're checking in this directory, this can't be the place for pre-generated schema.